packages/tools/pluggable-widgets-tools/scripts/e2e.js

```diff
@@ -128,26 +139,26 @@ async function exists(filePath) {
     }
 }
 
-async function getLatestMendixVersion() {
-    let latestMendixVersion;
+async function getMendixVersion() {
+    let mendixVersion;
+
+    if (process.env.MENDIX_VERSION) {
+        return process.env.MENDIX_VERSION;
+    }
     try {
-        const dockerTagsResponse = await fetch(
-            "https://registry.hub.docker.com/v1/repositories/mendix/runtime-base/tags"
+        const mendixVersions = await fetch(
+            "https://raw.githubusercontent.com/mendix/widgets-resources/master/configs/e2e/mendix-versions.json"
```

Comment:
Why do we keep track of the mendix version in master?

Reply:
To just rely in one place as master also contains mx8 latest version